cmd/minishift/cmd/start_test.go

```diff
@@ -384,6 +387,32 @@ func TestDetermineIsoUrlWithInvalidName(t *testing.T) {
 	determineIsoUrl("foo")
 }
 
+func Test_getslice(t *testing.T) {
```

Comment:
I feels like we are testing two things here, right. One retrieving a slice from the config and the other retrieving a slice from the command lines. Let's put this into two separate tests making the intention clearer